content/pages/guides/maintain-website-guide.md

```diff
@@ -8,6 +8,52 @@ You want to help with integrating new content into the site? Great to hear that!
 
 A new Coding Train video has been released and you want to add it to the website so everyone else can find it there? OK, just follow these steps:
 
+## Coding Challenges - starting from a template
+
+Data from the old website was scraped into starter templates that reside in a GitHub repo that can be found [here](https://github.com/skaser85/thecodingtrain_website_data).  Each template directory includes three items:
```

Comment:
```suggestion
Data from the old website was scraped into starter templates that [reside in a GitHub repo that can be found here](https://github.com/skaser85/thecodingtrain_website_data).  Each template directory includes three items:
```

Reply:
This is just a small note that having more description text in a link is helpful for accessibility / screen readers.

Reply:
Great point!  I'll keep that in mind in the future.